pkg/pki/pkcs7/pkcs7.go

```diff
@@ -211,5 +213,31 @@ func (k PublicKey) EmailAddresses() []string {
 
 // Subjects implements the pki.PublicKey interface
 func (k PublicKey) Subjects() []string {
-	return k.EmailAddresses()
+	// combine identities in the subject and SANs
+	identities := k.EmailAddresses()
+	cert, err := x509.ParseCertificate(k.rawCert)
```

Comment:
why not take `k.certs[0]`? Seems like that's the parsed k.rawCert: https://github.com/sigstore/rekor/blob/16915805c250dfc27108276d6598b14b177154e7/pkg/pki/pkcs7/pkcs7.go#L172

Reply:
done